OWNERS_ALIASES

```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-cluster-lifecycle-leads:
+    - lukemarsden
+    - luxas
+    - roberthbailey
+    - timothysc
+  cluster-api-admins:
+    - justinsb
```

Comment:
Justin was a kube-deploy admin for the aws image builder sub-project. Not sure if he cares about being an admin for this repo or not.